zebra-consensus/src/block/check.rs

```diff
@@ -121,17 +121,9 @@ pub fn subsidy_is_valid(block: &Block, network: Network) -> Result<(), BlockErro
     } else if halving_div.count_ones() != 1 {
         unreachable!("invalid halving divisor: the halving divisor must be a non-zero power of two")
     } else if height < canopy_activation_height {
-        // Founders rewards are paid up to Canopy activation, on both mainnet and testnet
-        let founders_reward = subsidy::founders_reward::founders_reward(height, network)
-            .expect("invalid Amount: founders reward should be valid");
-        let matching_values = subsidy::general::find_output_with_amount(coinbase, founders_reward);
-
-        // TODO: the exact founders reward value must be sent as a single output to the correct address
-        if !matching_values.is_empty() {
-            Ok(())
-        } else {
-            Err(SubsidyError::FoundersRewardNotFound)?
-        }
+        // Founders rewards are paid up to Canopy activation, on both mainnet and testnet.
+        // But we checkpoint in Canopy so founders reward does not apply for Zebra.
+        Ok(())
```

Comment:
```suggestion
        unreachable!("we cannot verify consensus rules before Canopy activation");
```
I don't know whether this will work, but it seems clearer to me if you don't actually verify blocks before Canopy.

Reply:
Yea, this is a good idea. It just needed to fix some test cases that were using older than Canopy blocks to make it work. I made this at  https://github.com/ZcashFoundation/zebra/pull/3430/commits/19bfa26c80d27777fca710f0e319665184e15367

Reply:
(I was just about to make this suggestion, thanks!)